src/test/shared/codelens/localLambdaRunner.test.ts

```diff
@@ -12,15 +12,14 @@ import { DebugConfiguration } from '../../../lambda/local/debugConfiguration'
 import * as localLambdaRunner from '../../../shared/codelens/localLambdaRunner'
 import * as fsUtils from '../../../shared/filesystemUtilities'
 import { ChildProcessResult } from '../../../shared/utilities/childProcess'
-import { ExtensionDisposableFiles } from '../../../shared/utilities/disposableFiles'
 import { FakeExtensionContext } from '../../fakeExtensionContext'
 import { FakeChannelLogger } from '../fakeChannelLogger'
 import { assertRejects } from '../utilities/assertUtils'
 
 describe('localLambdaRunner', async () => {
     let tempDir: string
     before(async () => {
-        await ExtensionDisposableFiles.initialize(new FakeExtensionContext())
+        FakeExtensionContext.getNew()
```

Comment:
I think this is preferable because it avoids adding a "clear" fucntion on `ExtensionDisposableFiles`, which would only be used by tests anyway.